it/controllers/GetBusinessDetailsControllerISpec.scala

```diff
@@ -52,36 +49,18 @@ class GetBusinessDetailsControllerISpec extends ComponentSpecBase {
             jsonBodyMatching(jsonSuccessOutput())
           )
         }
-        "return a valid IncomeSourceDetails model using IF platform" in {
-
-          isAuthorised(true)
-
-          And("I wiremock stub a successful getIncomeSourceDetails response")
-          BusinessDetailsCallWithNinoStub.stubGetIfBusinessDetails(testNino, incomeSourceDetailsSuccess)
-
-          When(s"I call GET /get-business-details/nino/$testNino")
-          val request = FakeRequest(controllers.routes.GetBusinessDetailsController.getBusinessDetails(testNino)).withHeaders("Authorization" -> "Bearer123")
```

Comment:
The new test doesn't have this request in it. Is this OK? Was it unneeded previously?

Reply:
Yeah. It was used for that special Test with submission stub which should had been deprecated. So, removed it.